### PR TITLE
Fix/iclr setup

### PR DESCRIPTION
**Pull request by melisabok**
- use tqdm for all the iterations.
- fix authorids field description.
- make the pdf field mandatory for the revision invitation

### CODE REVIEW

File: openreview/conference/templates/bidWebfield.js [hunk 1]
```diff
@@ -345,5 +321,58 @@ function addEdgesToNotes(validNotes, edgesMap) {
...
 // Go!
 main();
+
+// Event handlers
+$('#invitation-container').on('change', 'form.notes-search-form .score-dropdown', function(e) {
+  console.log('change!', $(this).val());
+  selectedScore = $(this).val();
+        return getPapersSortedByAffinity(0, selectedScore)
+        .then(function(notes) {
+          return renderNotes(prepareNotes(notes, conflictIds, bidsByNote));
```
Comment (melisabok) [inline openreview/conference/templates/bidWebfield.js:376]: replacing renderNotes with:

```
          $('.submissions-list', '#allPapers').replaceWith(
            Handlebars.templates['partials/noteList']({
              notes: prepareNotes(notes, conflictIds, bidsByNote),
              options: paperDisplayOptions
            })
          );
```

seems to fix the issues with the event but it doesn't reload the pagination bar, so it keeps in the same page. I'm not sure if we should reset the offset value to 0.

File: openreview/conference/templates/bidWebfield.js [hunk 2]
```diff
@@ -345,5 +250,127 @@ function addEdgesToNotes(validNotes, edgesMap) {
   return validNotes;
 }
 
+function updateNotes(notes) {
```
Comment (melisabok) [inline openreview/conference/templates/bidWebfield.js:253]: I fixed the issue with the tag widget events, I'm calling to re render the whole tab panel when the sorting criteria changes, it should be working now.

**Comment by zbialecki** [comment]
Does dev have the current webfield code?

**Comment by melisabok** [comment]
Thanks @zbialecki , I just deployed the changes in the dev site, could you take another look? I want to make sure the approach of reloading the whole tab panel is correct.

File: openreview/conference/templates/bidWebfield.js [hunk 3]
```diff
@@ -10,7 +10,35 @@ var BID_ID = '';
...
+  'High': [],
+  'Neutral': [],
+  'Very Low': [],
+  'Low': []
+};
+var sections = [];
+var scoresByName = {
+  'TPMS': 'thecvf.com/ECCV/2020/Conference/Reviewers/-/TPMS_Score',
+  'Affinity': 'thecvf.com/ECCV/2020/Conference/Reviewers/-/Affinity_Score'
+}
```
Comment (zbialecki) [inline openreview/conference/templates/bidWebfield.js:30]: I think this is fine for now, but I think we should come up with a more general method before this is merged into master.

**Comment by zbialecki** [comment]
I think it's working correctly, the only thing I noticed is that it's missing the referrer banner

**Review by zbialecki — APPROVED** [review]

**Review by celestemartinez — APPROVED** [review]

File: openreview/conference/templates/bidWebfield.js [hunk 4]
```diff
@@ -207,124 +211,19 @@ function renderContent(notes, conflictIds, bidEdges) {
...
-    sections[activeTab].active = true;
-
-    $('#notes .tabs-container').remove();
-
-    Webfield.ui.tabPanel(sections, {
-      container: '#notes',
-      hidden: true
+  $('#invitation-container').on('change', 'form.notes-search-form .score-dropdown', function(e) {
+    selectedScore = $(this).val();
+    return getPapersSortedByAffinity(0, selectedScore)
```
Comment (xkopenreview) [inline openreview/conference/templates/bidWebfield.js:216]: passing selectedScore to getPapersSortedByAffinity is not necessary as the function takes only 1 argument which is the offset

**Pull request merged**